ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml

```
@@ -0,0 +1,47 @@
+{{- if .Values.extendedMonitoring.certificates.exporterEnabled }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cert-exporter
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
```

```suggestion
{{- include "helm_lib_module_labels" (list .) | indent 2 }}
```
Just to keep the style. Or, we could switch to using `nindent` everywhere, but then there are a lot of changes to make.

---

I believe that using `nindent` will make the code more readable.
If you want to, I can prepare a PR to update it in other places.

---

solved in d9c4aa5b2dc24fa6f15b1ee5fb064c3ae6148dbc